src/modules/dual_path.py

```diff
@@ -482,3 +496,141 @@ def evaluate_network(self, set_names, top_down_language_activation=False):
                                      not has_wrong_det, f'"{line.message}"', ' '.join(entropy_idx),
                                      self.starting_epoch, has_cognate, has_false_friend])
                     logger.info(",".join(str(x) for x in log_info))
+
+
+    def evaluate_priming(self):
```

Comment:
There's quite some overlap with evaluate_network. In the future, I'd try to merge them/move the common parts to a separate function.